tests/network/bindingplugin_passt.go

```diff
@@ -331,5 +336,86 @@ EOL`, inetSuffix, serverIP, serverPort)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred())
+				vmi = libwait.WaitUntilVMIReady(vmi,
+					console.LoginToFedora,
+					libwait.WithFailOnWarnings(false),
+					libwait.WithTimeout(180),
+				)
+				return vmi
+			}
+
+			By("Starting a VMI")
+			migrateVMI := startPasstVMI(libvmi.NewFedora, console.LoginToFedora)
+			beforeMigNodeName := migrateVMI.Status.NodeName
+
+			By("Starting another VMI")
+			anotherVMI := startPasstVMI(libvmi.NewAlpine, console.LoginToAlpine)
+
+			By("Verify the VMIs can ping each other")
+			migrateVmiBeforeMigIP := libnet.GetVmiPrimaryIPByFamily(migrateVMI, ipFamily)
+			anotherVmiIP := libnet.GetVmiPrimaryIPByFamily(anotherVMI, ipFamily)
+			Expect(libnet.PingFromVMConsole(migrateVMI, anotherVmiIP)).To(Succeed())
+			Expect(libnet.PingFromVMConsole(anotherVMI, migrateVmiBeforeMigIP)).To(Succeed())
+
+			By("Perform migration")
+			migration := libmigration.New(migrateVMI.Name, migrateVMI.Namespace)
+			migrationUID := libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(kubevirt.Client(), migration)
+			migrateVMI = libmigration.ConfirmVMIPostMigration(kubevirt.Client(), migrateVMI, migrationUID)
+
+			By("Verify all the containers in the source pod were terminated")
+			labelSelector := fmt.Sprintf("%s=%s", v1.CreatedByLabel, string(migrateVMI.GetUID()))
+			fieldSelector := fmt.Sprintf("spec.nodeName==%s", beforeMigNodeName)
+
+			Eventually(func() bool {
+				pods, err := kubevirt.Client().CoreV1().Pods(migrateVMI.Namespace).List(context.Background(),
+					metav1.ListOptions{LabelSelector: labelSelector, FieldSelector: fieldSelector},
+				)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(len(pods.Items)).To(Equal(1))
+
+				return pods.Items[0].Status.Phase == k8sv1.PodSucceeded
+			}, 30*time.Second).Should(BeTrue())
+
+			By("Verify the VMI new IP is propogated to the status")
+			var migrateVmiAfterMigIP string
+			Eventually(func() bool {
+				migrateVMI, err = kubevirt.Client().VirtualMachineInstance(migrateVMI.Namespace).Get(context.Background(), migrateVMI.Name, &metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred(), "should have been able to retrive the VMI instance")
+				migrateVmiAfterMigIP = libnet.GetVmiPrimaryIPByFamily(migrateVMI, ipFamily)
+				return migrateVmiBeforeMigIP != migrateVmiAfterMigIP
+			}, 30*time.Second).Should(BeTrue(), "the VMI status should get a new IP after migration")
+
+			By("Verify the guest renewed the IP")
+			ipValidation := fmt.Sprintf("ip addr show eth0 | grep %s", migrateVmiAfterMigIP)
+			Eventually(func() error {
+				return console.RunCommand(migrateVMI, ipValidation, time.Second*5)
+			}, 30*time.Second).Should(Succeed())
+
+			By("Verify the VMIs can ping each other after migration")
+			Expect(libnet.PingFromVMConsole(migrateVMI, anotherVmiIP)).To(Succeed())
+			Expect(libnet.PingFromVMConsole(anotherVMI, migrateVmiAfterMigIP)).To(Succeed())
+		},
+			Entry("[IPv4]", k8sv1.IPv4Protocol),
+			Entry("[IPv6]", k8sv1.IPv6Protocol),
```

Comment:
All Our IPv4 IPv6 tests have this format. It was discussed years ago and we decided to take this approach and not testing IPv4 and IPv6 as part of the same test.
This decision may be revisited but I don't want to make the discussion about it on this PR. Until we decide differently, that's the standard.

Reply:
Ok, but I do not think we need such a large test coverage for this endge feature that is not even GA.
e2e tests are expensive and we should be careful and optimize as much as possible.

I think we need to be convinced it is a really needed and not the other way around (to convince it is not needed).

Reply:
I don't think it is an edge feature, it may replace masquerade and bridge binding in the future. I find it important to test it works with IPv6 out of the box as it is one extra point that passt has while masquerade doesn't.

Reply:
I am not saying we should not test IPv6, I am saying we should assume dual stack and test both through the same test.

Reply:
As I wrote in my first reply to the comment. That's the standard of how ipv4/ipv6 tests are tested. We had a long discussion about it in the past, and that's what we agreed to done. A work was done to change all the "dual-stack" tests to this format.
We can rethink this decision, but I prefer not to block this PR on it.